convert2rhel/main.py

```diff
@@ -100,6 +100,10 @@ def main():
         loggerinst.task("Final: rpm files modified by the conversion")
         systeminfo.system_info.modified_rpm_files_diff()
 
+        # Remove temporary folder with backup - last step of conversion
```

Comment:
Method and task is self-explanatory, the comment here is not needed. As a rule of thumb for comments, if something is not evidently clear by the way it's written and it cannot be refactored it might make sense to explain what it does. 

Likewise, comments explaining why we're doing something is almost always a good thing. This is most useful within docstrings to explain methods, but can be inline if the method is a bit larger and has a lot of side-effects.

Oh and comments can also indicate start and ends of certain things, like we do here in `main.py` with `# begin conversion process` etc.
```suggestion
```